### PR TITLE
Migrate moment to dayjs for Pages.Web

### PR DESCRIPTION
**Pull request by david-poindexter**
## Summary
Migrate `moment` to `dayjs` for `Pages.Web` project.  This relates to #3875.

### CODE REVIEW

File: Dnn.AdminExperience/ClientSide/Pages.Web/src/utils.js [hunk 1]
```diff
@@ -190,7 +190,9 @@ function formatDate(dateValue, longformat) {
         return "-";
     }
 
-    return Moment(dateValue).locale(utilities.getCulture()).format(longformat === true ? "LLL" : "L");
+    const localizedFormat = require('dayjs/plugin/localizedFormat');
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Pages.Web/src/utils.js:193]: Can we move this up near the imports

**Review by valadas — CHANGES_REQUESTED** [review]
Just a little `nit` for clarity :)

File: Dnn.AdminExperience/ClientSide/Pages.Web/src/utils.js (hunk 1 above)
Comment (david-poindexter) [reply Dnn.AdminExperience/ClientSide/Pages.Web/src/utils.js]: @valadas sure thing - I just pushed a new commit.

Comment (valadas) [reply Dnn.AdminExperience/ClientSide/Pages.Web/src/utils.js]: Thank you

**Review by valadas — APPROVED** [review]
Looks good to me.

**Review by mitchelsellers — APPROVED** [review]

**Pull request merged**